VRMKit.xcodeproj/project.pbxproj

```diff
@@ -930,7 +954,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
 				INFOPLIST_FILE = Example/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```

Comment:
I totally agree with end of support for iOS 10 👍

Reply:
Oh, I'm sorry. This target is an example app.
It’s okay to change VRMKit's deployment target. Of course, it's okay for another time. 👍

Reply:
I submitted another PR that obsolete older OS.
https://github.com/tattn/VRMKit/pull/10